app/javascript/store/modules/froggo_team/actions.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+import {
+    CREATE_NEW_FROGGO_TEAM,
+} from '../../action-types';
+
+export default {
+  [CREATE_NEW_FROGGO_TEAM](_,{ name, organization_id, users }) {
+    axios.post(
+      `/api/organizations/${organization_id}/froggo_teams`,
+      { name },
+      {headers: {"Content-Type": "application/json"} },
+    )
+    .then(async (response) => {
+        const { data: { id } } = response;
+        return Promise.all(users.map(user => axios.post(`/api/froggo_teams/${id}/add_member`, { github_login: user.login })))
+    })
```

Comment:
Expected indentation of 6 spaces but found 4                                                                   indent